frontend/src/components/DomainDetails.tsx

```diff
@@ -280,7 +280,7 @@ export const DomainDetails: React.FC<Props> = (props) => {
               <Accordion className={classes.accordion} key={vuln.id}>
                 <AccordionSummary expandIcon={<ExpandMore />}>
                   <Typography className={classes.accordionHeading}>
-                    {vuln.cve}
+                    {vuln.title}
```

Comment:
Is `vuln.title` equal to `vuln.cve` for CVE-related Vulnerabilities?

Reply:
Yes, it is.